test/integration/webhook/playground_test.go

```diff
@@ -69,6 +69,12 @@ var _ = ginkgo.Describe("Playground default and validation", func() {
 			},
 			failed: true,
 		}),
+		ginkgo.Entry("invalid BackendRuntime resources", &testValidatingCase{
```

Comment:
Could you add another example to cover the scenario: resources in **limits** but not in **requests** to make sure there's no panic.

Reply:
done